### DISCUSSION TITLE
Added quantummanager component functionality. enhancement

### DISCUSSION THREAD

**Discussion by fabrvers** [Quantum-Manager/tracker#182] [Q&A]
Hello, I'd like to make a few changes to your quantummanager component. 

1/ Add an option (show or hide the download zone) in scopescustom, ==> "dropareahidden" : "1" .

Example:
"scopescustom1: { "enable": "1", "id": "galerie-all", "title": "All galleries", "dropareahidden": "1", "path": "folder path" } 

2/ Add a field to enter a comment on a photo. We select a photo and in the right-hand column (meta-file-list) the metas are displayed, at this stage I want a <textarea> field to enter a comment, this comment will be saved for the selected photo. Can you tell me if it's possible to make these changes or give me the code to modify or insert?
![cap-comment1](https://github.com/Quantum-Manager/tracker/assets/73427164/320ed7ab-104d-486a-b9bc-99d0966f414c)
![cap-comment2](https://github.com/Quantum-Manager/tracker/assets/73427164/59737fb3-251d-4106-a460-db977ec7e529)

**Comment by ssnobben** [comment]
Did you get this to work?

**Reply by fabrvers** [reply]
no, if you have a solution I'm interested.
